RevenueCatUI/View+PurchaseRestoreCompleted.swift

```diff
@@ -24,6 +24,9 @@ public typealias PurchaseCompletedHandler = @MainActor @Sendable (_ transaction:
 /// A closure used for notifying of purchase initiation.
 public typealias PurchaseStartedHandler = @MainActor @Sendable () -> Void
```

Comment:
Same here, not sure if we can mark a typealias as deprecated 🤔

Reply:
I tried and it is possible but there's a big gotcha, which is that the warnings can't be silenced for types only for functions, or a least I couldn't get that to work and the api tests were warning all over the place

Reply:
Ok, then I think it's fine to just deprecate the usages 👍

Reply:
managed to add it in [f5c21b5](https://github.com/RevenueCat/purchases-ios/pull/3693/commits/f5c21b56a8baf64ea314e43f54f87908602d52fb) and [e5b935f](https://github.com/RevenueCat/purchases-ios/pull/3693/commits/e5b935fe84a7bf111dc838fc5a6a76d5cef7baf1)